mne/utils/tests/test_testing.py

```diff
@@ -4,7 +4,25 @@
 import pytest
 
 from mne.datasets import testing
+from mne.utils._testing import _click_ch_name
 from mne.utils import (_TempDir, _url_to_local_path, buggy_mkl_svd)
+from mne.io import read_raw_fif
+
+base_dir = op.join(op.dirname(__file__), '..', '..', 'io', 'tests', 'data')
+raw_fname = op.join(base_dir, 'test_raw.fif')
+
+
+def test_click_ch_name():
+    """Test fake clicks."""
+    import matplotlib
+    import matplotlib.pyplot as plt
+    matplotlib.use('Agg')
```

Comment:
calls to `matplotlib.use` need to happen before you import pyplot.

Reply:
Then you get a style error that imports aren't at the top I thought. That's copied from `mne_bids.tests.test_inspect.py`. Maybe I'm wrong though.

Reply:
it used to be the case that importing pyplot sets the backend under the hood if it's not set already, and that it couldn't be changed by subsequent calls to `matplotlib.use`.  It's possible that that's changed by now.  If not, you'd need to use an appropriate `noqa` comment to work around the style error.

Reply:
I believe that has been changed but again could be wrong, but I used to do the noqa work around and now I don't and things seem okay